Tour_Booking/tour_booking/views.py

```diff
@@ -1,11 +1,15 @@
 from django.shortcuts import render, redirect
 from django.utils.translation import gettext
+from django.utils import timezone
 from django.views import generic
-from .models import Tour
+from .models import Tour, Booking
 from django.contrib.auth import authenticate, login, logout
+from django.contrib.auth.decorators import login_required
 from django.db.models import Q
-from .models import Tour
+from django.urls import reverse
 from .forms import TourSearchForm
+from .forms import BookingForm
```

Comment:
Gom code lại cho gọn nha em
```suggestion
from .forms import TourSearchForm, BookingForm
```